scripts/process-spectrum-postcss-plugin.js

```diff
@@ -662,12 +662,13 @@ class ComponentConfig {
                             throw new Error(message);
                         }
                     }
+                    const operator = attribute.wildcard ? '*=' : '=';
```

Comment:
@Westbrook Here's where I added the `wildcard` option for enum attribute conversion in CSS processing. This is better...